assets/css/_diversion_page.scss

```diff
@@ -161,3 +165,14 @@
   height: 1.5rem;
   width: 1.5rem;
 }
+
+.c-diversion-panel__outline-button {
+  height: 2rem;
+  border-radius: 0.2rem;
+  font-size: 0.875rem;
+  padding: 0.375rem 0.5rem;
+
+  &--copy {
+    margin: 0.78125rem 0;
+  }
```

Comment:
I think this margin should be handled via the `.align-items-baseline` class on the parent flex